docs/FAQ/jwe-usage.md

```
@@ -0,0 +1,139 @@
+
+Encrypt userinfo responses in eSignet using JWE ([RFC 7516](https://datatracker.ietf.org/doc/html/rfc7516)), ensuring sensitive user data is only readable by the intended relying party.
+
+---
+
+## Table of Contents
+
+- [Background](#background)
+- [How It Works](#how-it-works)
+- [Prerequisites](#prerequisites)
+- [Configuration](#configuration)
+  - [1. Set Encryption Public Key](#1-set-encryption-public-key)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)